Show latest instance by default (#8973)
cloudflare/workers-sdk: .changeset/yummy-dolls-rescue.md

---

> "wrangler": patch

Reviewer: also i suppose this is technically a feature not a fix, so this should be a minor

Author: Alright, closing the backport PR then, since it won't be needed :)

---

> Show latest instance by dafault on workflows describe command

Reviewer: ```suggestion
Show latest instance by dafault on `workflows instances describe` command
```

> Show latest instance by dafault on `workflows isntances describe` command

Reviewer 2: ```suggestion
Show latest instance by default on `workflows instances describe` command
```

> Show latest instance by dafault on `workflows instances describe` command

Reviewer 2: ```suggestion
Show latest instance by default on `workflows instances describe` command
```

> Show latest instance by dafault on `workflows instances describe` command

Reviewer 3: ```suggestion
Show latest instance by default on `workflows instances describe` command
```
